model/metrics/gen-ai.yaml

```diff
@@ -0,0 +1,33 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'LLM attributes'
+    attributes:
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in an error"
+      - ref: server.address
+        requirement_level: required
+      - ref: gen_ai.response.model
+        requirement_level: required
+      - ref: gen_ai.system
+        requirement_level: required
+  - id: metric.gen_ai.tokens.usage
+    type: metric
+    metric_name: gen_ai.tokens.usage
+    brief: 'LLM usage attributes'
+    instrument: histogram
+    unit: "tokens"
+    stability: experimental
+    extends: metric_attributes.gen_ai
+    attributes:
+      - ref: gen_ai.usage.token_type
+        requirement_level: required
+  - id: metric.gen_ai.request.duration
+    type: metric
+    metric_name: gen_ai.request.duration
```

Comment:
I'd like to explore `gen_ai.operation.duration`

Reply:
I made this change and let's see what further feedback we get.